mos-platform/mega65/basic-header.S

```diff
@@ -0,0 +1,10 @@
+; MEGA65 BASIC header that jumps to _start.
+
+.section .basic_header,"aR",@progbits
+#	.short next_line	; address of next BASIC line
+        .byte 0x0a, 0x20, 0x0a, 0x00, 0xfe, 0x02, 0x20, 0x30, 0x00 ; 10 BANK 0
+        .byte 0x15, 0x20, 0x14, 0x00, 0x9e, 0x20 ; 20 SYS
+	.mos_addr_asciz _start, 4	; the location of the _start symbol, in 4 decimal
```

Comment:
is there a more elegant way of doing this?

Reply:
Not that I know of; this seems pretty idiomatic for BASIC headers.